### PR TITLE
Heedls 792 add prn

### PR DESCRIPTION
**Pull request by JBsoftwire**
### JIRA link
_[HEEDLS-792](https://softwiretech.atlassian.net/browse/HEEDLS-792)_

### Description
A fields for Professional Registration Number has been added to `_SearchableDelegateCard.cshtml`, `_GroupDelegateCard`, `_SearchableCourseDelegateCard.cshtml` and `_unapprovedDelegateExpandable.cshtml`, as well as backend models and database queries required to deliver the information.

### Screenshots
image
image
image
image

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors.
- [x] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [x] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateApprovals/_unapprovedDelegateExpandable.cshtml [hunk 1]
```diff
@@ -34,7 +34,14 @@
...
         </div>
-        @foreach (var customPrompt in Model.CustomPrompts) {
+        <div class="nhsuk-summary-list__row basic-summary-list__row">
+          <dt class="nhsuk-summary-list__key">
+            Professional Registration Number
+          </dt>
+          <partial name="_SummaryFieldValue" model="@Model.ProfessionalRegistrationNumber" />
+        </div>
+        @foreach (var customPrompt in Model.CustomPrompts)
+        {
```
Comment (DanBloxham-sw) [inline DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateApprovals/_unapprovedDelegateExpandable.cshtml:44]: Minor, looks like the formatter needs to be re-run as the open brace has been shifted down when it should be like it was before.

File: DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/Shared/DelegateInfoViewModel.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@ public DelegateInfoViewModel(DelegateUserCard delegateUser, IEnumerable<CustomFi
             Id = delegateUser.Id;
             Name = delegateUser.SearchableName;
             CandidateNumber = delegateUser.CandidateNumber;
-            
+
```
Comment (DanBloxham-sw) [inline DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/Shared/DelegateInfoViewModel.cs:17]: Empty line?

**Review by DanBloxham-sw — CHANGES_REQUESTED** [review]
Couple of very minor formatting changes, but otherwise good.

**Review by AlexJacksonDS — COMMENTED** [review]
Looks good to me, just the minor things Dan noted

File: DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/Shared/DelegateInfoViewModel.cs (hunk 2 above)
Comment (JBsoftwire) [reply DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/Shared/DelegateInfoViewModel.cs:17]: It sections the properties for readability. I assume the change to it is something the auto-formatter did.

**Review by AlexJacksonDS — APPROVED** [review]

**Review by DanBloxham-sw — APPROVED** [review]

**Pull request merged**